servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpServerBuilder.java

```diff
@@ -356,11 +349,13 @@ private HttpExecutionContext buildExecutionContext(final HttpExecutionStrategy s
                 filteredService, drainRequestPayloadBody);
     }
 
-    private HttpExecutionStrategy computeServiceStrategy(Object service) {
-        HttpExecutionStrategy serviceStrategy = requiredOffloads(service, defaultStrategy());
+    private HttpExecutionStrategy computeServiceStrategy(HttpExecutionStrategyInfluencer service) {
+        HttpExecutionStrategy serviceStrategy = service.requiredOffloads();
         HttpExecutionStrategy filterStrategy = computeRequiredStrategy(serviceFilters, serviceStrategy);
-        return defaultStrategy() == strategy ? filterStrategy :
-                strategy.hasOffloads() ? strategy.merge(filterStrategy) : offloadNone();
+        return defaultStrategy() == strategy ?
+            filterStrategy.hasOffloads() ? strategy.merge(filterStrategy) : strategy :
```

Comment:
`strategy.merge(filterStrategy)` - this will result in a warning log printed by `DEFAULT_HTTP_EXECUTION_STRATEGY`. Should it be `filterStrategy.hasOffloads() ? filterStrategy : strategy`?

Reply:
yes, well spotted. this can be just `filterStrategy`.